l10n/en/firefox/whatsnew/whatsnew-s2d.ftl

```diff
@@ -0,0 +1,16 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/whatsnew/all/
+
+whatsnew-s2d-download-firefox-for-android = Download { -brand-name-firefox } for { -brand-name-android } and { -brand-name-ios }
+whatsnew-s2d-want-privacy-on-every-device = Want privacy on every device?
+
+# "You got it" here is a casual answer to the previous question, "Want privacy on every device?"
+whatsnew-s2d-you-got-it-get-firefox-for = You got it. Get { -brand-name-firefox } for mobile.
+
+# Line break is for visual formatting only
+whatsnew-s2d-send-firefox-to-your-phone = { -brand-name-send } { -brand-name-firefox } to your phone<br> and unleash your Internet.
```

Comment:
I missed this one `Send` should be a verb, not a product name in this instance. Please submit the PR to www-l10n

Reply:
Good catch, will update